libraries/chain/apply_context.cpp

```diff
@@ -882,6 +983,10 @@ void apply_context::add_ram_usage( account_name account, int64_t ram_delta ) {
    }
 }
 
+void apply_context::add_disk_usage( account_name account, int64_t disk_delta ) {
+   trx_context.add_disk_usage( account, disk_delta );
```

Comment:
Are we not interested in tracking DISK deltas per account in the traces just like with RAM?

Reply:
I saw you added the `account_disk_deltas` to `action_trace`, but it will also need to be included into a SHiP `action_trace_v1` struct. Note that we will need to introduce an `action_trace_v1` struct here but it will also need to be included for the changes planned for action return values (so watch out for conflicts with that upcoming PR). The current implementation of action return values in develop (which this PR branch includes) introduced `action_receipt_v1`, because the entire return value was added into the receipt (which we don't want to do anymore), but did not need to modify the `action_trace`. The new approach will interestingly not require a change to the `action_receipt` (there is no need for `action_receipt_v1` because there is no new field added to the receipt, instead the way `act_digest` gets computed changes when the `ACTION_RETURN_VALUE` protocol feature is activated) but will require an `action_trace_v1` to hold the `return_value` field.